contracts/strategies/strategy.sol.example

```diff
@@ -0,0 +1,125 @@
+/// @notice SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+/// @dev Import the appropriate base contract
+import "../strategy-joe-rush-farm-base.sol";
+
+/// @dev Contracts use NatSpec Format comments. NatSpec documentation is located here: https://docs.soliditylang.org/en/develop/natspec-format.html
+/// @notice The strategy contract for TraderJoe's Token1/Token2 Liquidity Pool with double rewards
+/// @dev If one of the tokens is a bridge extension token (ends with ".e"), the trailing E should be capitalized like so: StrategyJoeAvaxDaiE
+contract StrategyJoeToken1Token2 is StrategyJoeRushFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public token1_token2_poolId = 10;
+    address public joe_token1_token2_lp = 0x1000000000;
+    
+    /// @dev Addresses for tokens not found in the base contracts
+    address public token2 = 0x2000000000;
+    address public reward1 = 0x300000000;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyJoeRushFarmBase(
+            token1_token2_poolId,
+            joe_token1_token2_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    /// @dev Collect token fees and add liquidity to base pair
+    function harvest() public override onlyBenevolent {
+        /// @dev Ensure the deposit function interfaces with the correct controller for your platform (Joe, Pangolin, etc.)
```

Comment:
The dev tag is not needed here since we are not in a natspec header.  But this comment is definitely useful, as it describes why we are doing what we are doing.